### ISSUE TITLE
Running `miral-desktop` in a terminal window on 16.04 LTS starts the shell under X11

### ISSUE THREAD

**Issue by AlanGriffiths** [MirServer/mir#428]
Encountered whilst testing 0.32 release candidate (#406)

**Comment by AlanGriffiths** [comment]
@RAOF would you please check what's going on here?

It seems to be 16.04 specific: I don't see this on 17.10 or 18.04.

**Comment by AlanGriffiths** [comment]
FWIW trying to force "mesa-kms" doesn't work:
````
$ miral-desktop --platform-graphics-lib /usr/lib/x86_64-linux-gnu/mir/server-platform/graphics-mesa-kms.so.15
waiting for /run/user/1000/miral_socket
[2018-06-26 17:34:22.547755] mirserver: Starting
[2018-06-26 17:34:22.549728] mirserver: Selected driver: mir:mesa-kms (version 0.32.0)
[2018-06-26 17:34:22.568079] <DEBUG> mirserver: Using logind for session management
[2018-06-26 17:34:22.571629] <WARNING> mesa-kms: Failed to open DRM device node /dev/dri/card0: 0 (Success)
ERROR: /build/mir-VekrfY/mir-0.32.0~rc9-g29632f2fcc/src/server/graphics/default_configuration.cpp(172): Throw in function mir::DefaultServerConfiguration::the_graphics_platform()::<lambda()>
Dynamic exception type: boost::exception_detail::clone_impl<boost::exception_detail::error_info_injector<std::runtime_error> >
std::exception::what: Exception while creating graphics platform
ERROR: /build/mir-VekrfY/mir-0.32.0~rc9-g29632f2fcc/src/platforms/mesa/server/display_helpers.cpp(111): Throw in function static std::vector<std::shared_ptr<mir::graphics::mesa::helpers::DRMHelper> > mir::graphics::mesa::helpers::DRMHelper::open_all_devices(const std::shared_ptr<mir::udev::Context>&, mir::ConsoleServices&)
Dynamic exception type: boost::exception_detail::clone_impl<boost::exception_detail::error_info_injector<std::system_error> >
std::exception::what: Error opening DRM device: Success
```

**Comment by RAOF** [comment]
Raaaargh! Setting up a 16.04 test environment is going poorly :(

**Comment by AlanGriffiths** [comment]
Fixed by #501
